experimental/PropertyDDS/packages/property-binder/src/data_binder/data_binding_registry.ts

```diff
@@ -143,7 +143,7 @@ class DataBindingRegistry {
 
     // BFS of the templates and return the first registered one (closest)
     while (typeidQueue.length > 0) {
-      const currentTypeid = typeidQueue.shift();
+      const currentTypeid = typeidQueue.shift() as string;
```

Comment:
Nit: If the cast is because 'shift()' can return undefined for an empty array, a non-null type assertion would be clearer, even if it ESLint flags non-null assertions:
```ts
// eslint-disable-next-line: no-non-null-assertion -- '.length > 0' ensures 'shift()' does not return undefined.
... = typeidQueue.shift()!;
```